### PR TITLE
add general purpose text detection models

### PR DESCRIPTION
**Pull request by hadipash**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

Added DBNet and DBNet++ general purpose models trained on large-scale dataset with two primary languages: Chinese and English.

## Related Issues and PRs

#222: [630][Model] DBNet中文数据集支持，中英文混合训练.

### CODE REVIEW

**Comment by hadipash** [comment]
@SamitHuang @HaoyangLee please review this PR. I will re-run CI build once DBNet++ weights are uploaded to the server.

File: configs/det/dbnet/README.md [hunk 1]
```diff
@@ -56,9 +56,26 @@ combination of these two modules leads to scale-robust feature fusion.
...
+
+<div align="center">
+
+| **Model** | **Context**    | **Backbone** | **Languages**     | **F-score on Our Test Set** | **Throughput** | **Download**                                                                                             |
+|-----------|----------------|--------------|-------------------|:---------------------------:|----------------|----------------------------------------------------------------------------------------------------------|
+| DBNet     | D910x8-MS2.0-G | ResNet-50    | Chinese + English |           83.41%            | 256 img/s      | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_ch_en_general-a5dbb141.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_ch_en_general-a5dbb141-912f0a90.mindir) |
+| DBNet++   | D910x4-MS2.0-G | ResNet-50    | Chinese + English |           84.30%            | 256 img/s      | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnetpp_resnet50_ch_en_general-884ba5b9.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnetpp_resnet50_ch_en_general-884ba5b9-b3f52398.mindir) |
+</div>
```
Comment (HaoyangLee) [inline configs/det/dbnet/README.md:73]: Please specify the data shape when exporting mindir below this table, like the ICDAR2015 one.

**Review by HaoyangLee — APPROVED** [review]

**Comment by HaoyangLee** [comment]
The latest `docs/en/inference/model_list.md` add two column: F-score and FPS. You can leave them blank for the general dbnet if you don't have these info now.

**Comment by hadipash** [comment]
> The latest `docs/en/inference/model_list.md` add two column: F-score and FPS. You can leave them blank for the general dbnet if you don't have these info now.

Done.

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**